### DISCUSSION TITLE
Recognize App missing models

### DISCUSSION THREAD

**Discussion by tuetenk0pp** [nextcloud/all-in-one#3791] [Questions] [answered]
I installed the recognize app algong with the memories app in my aio and ran `occ recognize:download-models` to download the models. But after a few days I noticed that the recognize app was no longer working and I had to run the command again. I assume that aio recreated the container in the meantime.

**Comment by szaimen** [comment, marked as answer]
Hi, this is most likely an issue within the recognize app and should be reported here: https://github.com/nextcloud/recognize/issues

**Reply by tuetenk0pp** [reply]
I also have a "normal" nextcloud instance with the same apps installed and it works just fine. That's why I started the discussion here

**Reply by tuetenk0pp** [reply]
[nextcloud/recognize/#1036](https://github.com/nextcloud/recognize/issues/1036)
